mix.exs

```diff
@@ -88,7 +95,12 @@ defmodule Plenario.Mixfile do
       # in Plug and Phoenix based applications.
       #
       # MIT
-      {:explode, "~> 1.0.0"}
+      {:explode, "~> 1.0.0"},
+
+      # Simple, robust temporary file support for Elixir.
```

Comment:
I like including the license, but I'd rather have a short dry description like "utility for temporary file handling" than the package's own tagline